packages/rainbowkit/src/components/ChainModal/ChainModal.test.tsx

```diff
@@ -1,3 +1,4 @@
+import '@testing-library/jest-dom';
```

Comment:
I noticed we should usually import this in the test setup file (so that is available to all testable components). Is there a reason we can't do that?

Reply:
Ah yeah good point. Added back `setupFiles` to vitest.config.ts